lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.ts

```diff
@@ -148,6 +171,7 @@ export class ContentMetadataComponent implements OnChanges, OnInit, OnDestroy {
     private targetProperty: CardViewBaseItemModel;
     private classifiableChangedSubject = new Subject<void>();
     private _saving = false;
+    private readonly destroyRef = inject(DestroyRef);
```

Comment:
can you keep private and private readonly separately? (please add empty line between them)